#30 member overview

emanuelen5 opened the pull request:
# Changes
Adding a menu specifically for key handouts.

# Demo
![Showing how to use it](https://user-images.githubusercontent.com/7572427/158034960-b0ae6d4d-a432-4b5d-8af9-2e391f73d2f7.gif)

# Todo
* [X] Extra fields
* [x] Pending orders

andersroos commented on admin/src/Membership/MemberBoxOverview.jsx:
```diff
@@ -0,0 +1,23 @@
+// };
+//
+//
+// export default MemberBoxOverview;
```
Remove file if unused.

emanuelen5 commented:
New version.

## Changes
* Same save-button for both key and other fields
* Adding fields for email and phone number
* Some bug fixes

## Demonstration
![Animation](https://user-images.githubusercontent.com/7572427/158483239-70872cb1-2980-450c-a272-0aea33f75823.gif)

emanuelen5 commented:
## Changes
* Spans visualized
* Added zipcode

## Screenshot
![image](https://user-images.githubusercontent.com/7572427/158708837-d32f23a0-b9d0-4df7-a0c4-57e91e0061b0.png)

emanuelen5 commented:
## Changes
* Improved margins (left of sections, and between paragraphs)
* Fixed alignment of some input fields
* Fixed formatting of input fields to look like others

## Screenshot
![image](https://user-images.githubusercontent.com/7572427/158908938-4895434a-4580-4447-be6c-cdfc0be02e0d.png)

emanuelen5 commented:
Closes #30
